scale/utils.py

```diff
@@ -0,0 +1,130 @@
+#/usr/bin/python3
+
+"""Kubernetes API access functions"""
+
+from kubernetes import client, config
+#import requests
+import logging
+
+scale_logger = logging.getLogger("scale")
+logging.getLogger("kubernetes").setLevel(logging.WARNING)
+config.load_kube_config()
+v1 = client.CoreV1Api()
+
+
+# def generate_url(host, port):
+#    return "http://" + host + ':' + port + "/api/v1/"
+
+
+def get_nodes():
+    """Return a list of v1.Node"""
+    scale_logger.debug("Getting all nodes in the cluster")
+    return v1.list_node().items
+#     r = requests.get(generate_url(API_HOST, API_PORT) +
```

Comment:
Get rid of commented out code :) If you do that all as a single commit, we can revert that later if we want to!